src/app/frank/fd_frank_main.c

```diff
@@ -115,6 +119,13 @@ main( int     argc,
 
   } while(0);
 
+  #ifdef FD_HAS_SANDBOX
+  fd_sandbox_profile_t profile;
+  fd_sandbox_profile_init( &profile );
+
+  fd_sandbox(&profile);
+  #endif /* FD_HAS_SANDBOX */
+
```

Comment:
I don't think that we want to automatically sandbox everything from the start. The initialization might require more permissions as it will do things such as mapping other workspaces or setting up xdp. We could probably have two sandboxing steps (pre-init, post-init), but as long as we can consider init "safe", we can just live with post-init sandboxing.